macros/iam/login-member.mdx

```diff
@@ -2,14 +2,10 @@
 macro: login-member
 ---
 
-<Message type="important">
-  Login via [Single Sign-On (SSO)](/account/concepts/#single-sign-on-sso) is currently not available for Members.
-</Message>
-
 1. Open your web browser and go to the [Scaleway console](https://console.scaleway.com).
 2. Click [Log in as an IAM Member](https://account.scaleway.com/member-login?redirectToUrl=https://console.scaleway.com/).
     <Message type="important">
-      Logging into an Organization as a Member is different from logging in as a Guest or as an Owner. To perform a non-Member login, you can follow the classic [How to log into the console](/account/how-to/log-in-to-the-console) procedure.
+      Logging into an Organization as a Member is different from logging in as an Owner. To perform a non-Member login, you can follow the classic [How to log into the console](/account/how-to/log-in-to-the-console) procedure.
```

Comment:
@ldecarvalho-doc I am not sure we should refer to Owner login as "Classic". Can we have another wording?

Reply:
we can just remove the word
```suggestion
      Logging into an Organization as a Member is different from logging in as an Owner. To perform a non-Member login, you can follow the procedure described in the [How to log into the console](/account/how-to/log-in-to-the-console) documentation page.
```